app/models/project/pdf_export/project_initiation.rb

```diff
@@ -0,0 +1,202 @@
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+class Project::PDFExport::ProjectInitiation < Exports::Exporter
+  include Exports::PDF::Common::Common
+  include Exports::PDF::Common::Attachments
+  include Exports::PDF::Common::Logo
+  include Exports::PDF::Common::Macro
+  include Exports::PDF::Common::Markdown
+  include Exports::PDF::Components::Page
+  include Exports::PDF::Components::Cover
+  include Project::PDFExport::Common::ProjectAttributes
+  include Project::PDFExport::ProjectInitiation::Styles
+
+  attr_accessor :pdf
+
+  self.model = Project
+
+  alias :project :object
+
+  def self.key
+    :project_initiation_export_pdf
+  end
+
+  def initialize(project, _options = {})
+    super
+
+    @page_count = 0
+    setup_page!
+  end
+
+  def setup_page!
+    self.pdf = get_pdf
+    configure_page_size!(:portrait)
+    pdf.title = heading
+  end
+
+  def export!
+    render_doc
+    success(pdf.render)
+  rescue StandardError => e
+    Rails.logger.error "Failed to generate project creation PDF:  #{e.message}:\n#{e.backtrace.join("\n")}"
+    error(I18n.t(:error_pdf_failed_to_export, error: e.message))
+  ensure
+    delete_all_resized_images
+  end
+
+  def render_doc
+    render_project_initiation
+    render_again_with_total_page_nrs
+  end
+
+  def render_again_with_total_page_nrs
+    @total_page_nr = pdf.page_count + @page_count
+    @page_count = 0
+    setup_page! # clear current pdf
+    render_project_initiation
+  end
+
+  def render_project_initiation
+    write_cover_page! if with_cover?
+    write_title!
+    write_project_initiation
+    write_headers_footers
+  end
+
+  def write_headers_footers
+    write_logo!
+    write_footers!
+  end
+
+  def export_datetime
+    @export_datetime = Time.zone.now
+  end
+
+  def footer_date
+    format_time(export_datetime)
+  end
+
+  def cover_page_dates
+    nil
+  end
+
+  def cover_page_subheading
+    nil
+  end
+
+  def cover_page_title
+    project.name
+  end
+
+  def cover_page_heading
+    heading
+  end
+
+  def heading
+    I18n.t(:"export.project_initiation.title")
```

Comment:
Please note, that the name of the wizard and by that the name of the work package and I think in turn the name of this heading will become configurable. This is just a heads up for now.